spec/controllers/v2/concerns/tailoring_file_spec.rb

```diff
@@ -0,0 +1,208 @@
+# frozen_string_literal: true
```

Comment:
Tailoring file is not a concern, so this is semantically not correct. You can have a shared example inside a regular test. Then I think you can deduplicate a bunch of things, like you would not even need to specify the `parent`